### ISSUE TITLE
Primary game stream logic working incorrectly

### ISSUE THREAD

**Issue by WHiT3R4bBiT** [ahgl/ahgl-site#137]
Under Home -> Api -> Games -> [Game] channels section, there's a checkbox for 'Primary'.

Right now, a channel's stream will only appear if and only if that stream is live and it is marked as primary  while no others are marked as primary (nomatter if they're offline).

Desired behavior is that all channels marked as primary will be checked for being live first. If one or more primary channels are live, they're displayed. If not, the non-primary channels are checked for being live and one is displayed if live.

**Comment by Dimitri-Gnidash** [comment]
` If one or more primary channels are live, they're displayed` @WHiT3R4bBiT How do we display multiple channels?

**Comment by WHiT3R4bBiT** [comment]
Only display one of the available channels. Preferably at random on page load although it doesn't have to be random.

**Comment by WHiT3R4bBiT** [comment]
As an update to current behavior after testing - it appears multiple channels can be marked and shown as primary, but only the first to get the primary mark acts like the primary and will show as a stream (even if it's offline, in which case no stream will show)

**Comment by Dimitri-Gnidash** [comment]
Yes, this does not look to bad - I have looked in the code and we should have it done in the next few days.
